tools/ports/sdl2_mixer.py

```diff
@@ -52,7 +86,10 @@ def clear(ports, settings, shared):
 
 def process_dependencies(settings):
   settings.USE_SDL = 2
-  settings.USE_VORBIS = 1
+  if "ogg" in settings.SDL2_MIXER_FORMATS:
+    settings.USE_VORBIS = 1
+  if "mp3" in settings.SDL2_MIXER_FORMATS:
+    settings.USE_MPG123 = 1
```

Comment:
You can conditionally add to deps here (see `tools/ports/sdl2_image.py`)

Reply:
:sparkles: